### ISSUE TITLE
1.18.2 structures won't spawn in rf dimensions

### ISSUE THREAD

**Issue by NikoTheGreat** [McJty/RFToolsDimensions#414]
I cannot get structures such as nether fortresses to spawn. I have attempted many different combinations of dimlets to generate a nether fortress or other structure but they won't spawn in my world. I'm not sure if this is because of my computer itself or what. The structures will spawn in the default Minecraft dimensions but not in rf dimensions. I was able to get ruined nether portals to work but no other structures.

Steps to reproduce: 

Open a world in creative mode and attempt to make a dimension with the fortress dimlet or other structure. (Usually, I use a void world because structures are easier to find in void worlds). Fly around and there are no spawned structures.

Versions:
* Minecraft: 1.18.2
* Forge: 40.1.73
* McJtyLib: 1.18-6.0.16
* RFTools: 1.18-3.0.10
* RFTools Dimensions: 1.18-8.08

No other mods are installed.

Thank you so much for your time!   =)

**Comment by McJty** [comment]
You have TerraBlender mod? If so, that's the reason. It has a bug that prevents features from other mods to spawn in rftools dimensions. Author doesn't work on 1.18 anymore but in 1.19 the bug is fixed

**Comment by NikoTheGreat** [comment]
I don't have the TerraBlender mod installed. I uninstalled all other mods other than the RF series mods and McJtyLib to test the world generation and it still doesn't work in 1.18.2. I was able to generate a nether fortress in 1.12.2, tomorrow I'll download the 1.16 version and test that out as well.

**Comment by NikoTheGreat** [comment]
I tried it out in 1.16.5, but I couldn't find any of the structure dimlets in the dimlet workbench or spawn them in manually which is really weird. It's almost as if they were removed from the 1.16.5 version of RF tools dimensions. I will try to get it to work on a fresh install of Minecraft on a separate computer to see if I can replicate the issue or see if it has something to do with my hardware or installation.


Below is a screenshot of me searching for the fortress dimlet in the 1.16.5 dimlet workbench. None of the other structure dimlets would show up either.
![Screenshot 2022-08-25 140544](https://user-images.githubusercontent.com/16003271/186743066-03421369-a16d-401b-97cb-9312430e4f47.png)



Below is a screenshot of a nether fortress generated in an RF tools void world on Minecraft 1.12.2 world.
![Screenshot 2022-08-25 143651](https://user-images.githubusercontent.com/16003271/186742817-17de9224-24f5-4173-87a5-1dca1a4ef772.png)

**Comment by NikoTheGreat** [comment]
I did a fresh install on another computer and downloaded the required mods. After flying around for two Minecraft days and nights I couldn't find any structures other than the RF tools dimlet structure. My guess is that Minecraft changed the way structures are spawned so it doesn't work in RF tools dimensions. Either that or I'm just ridiculously unlucky. I tried using the locate structure command, but that didn't work either. Do you know if anyone else is having this problem?

**Comment by Johnny2525** [comment]
I am having the same issue on 1.18.2 the structures don't spawn.
I tried spawning a monument but in the place where the monument should spawn I only see guardians but no structure.
It seems like it actually tried spawning it (or spawned it) but for some reason it was overwritten by the world generator
[ I also don't have TerraBlender installed but I am playing on a SkyBlock modpack ]

![image](https://user-images.githubusercontent.com/4202240/187069590-17b1d055-758c-41cf-a4e1-511774fecdad.png)

![image](https://user-images.githubusercontent.com/4202240/187069598-6801aa42-8570-4720-8fc4-8df7a0c58575.png)

**Comment by NikoTheGreat** [comment]
I got the same thing to happen in my world. I found wither skeletons, but no fortress. I don't think this issue is happening because of other mods, in my previous testing I uninstalled everything but the RF mods, and McJtyLib. I didn't notice any difference after adding back my other mods. It is weird that the structure-specific mobs still spawn so there definitely should be a structure there.


Wither skeletons without a nether fortress in my RF dimension:
![Screenshot 2022-08-28 141523](https://user-images.githubusercontent.com/16003271/187089094-b1d2940d-a0e8-40e7-8764-1713c3c619e8.png)
